src/main/java/ch/njol/skript/SkriptConfig.java

```diff
@@ -50,230 +34,185 @@
+
+	public final static Option<Boolean> enableEffectCommands = new Option<>("enable effect commands", false);
+	public final static Option<String> effectCommandToken = new Option<>("effect command token", "!");
+	public final static Option<Boolean> allowOpsToUseEffectCommands = new Option<>("allow ops to use effect commands", false);
+
 	// everything handled by Variables
 	public final static OptionSection databases = new OptionSection("databases");
-	
-	public final static Option<Boolean> usePlayerUUIDsInVariableNames = new Option<Boolean>("use player UUIDs in variable names", false); // TODO change to true later (as well as in the default config)
-	public final static Option<Boolean> enablePlayerVariableFix = new Option<Boolean>("player variable fix", true);
-	
+
+	public final static Option<Boolean> usePlayerUUIDsInVariableNames = new Option<>("use player UUIDs in variable names", false); // TODO change to true later (as well as in the default config)
+	public final static Option<Boolean> enablePlayerVariableFix = new Option<>("player variable fix", true);
+
 	@SuppressWarnings("null")
 	private final static DateFormat shortDateFormat = DateFormat.getDateTimeInstance(DateFormat.SHORT, DateFormat.SHORT);
-	private final static Option<DateFormat> dateFormat = new Option<DateFormat>("date format", shortDateFormat, new Converter<String, DateFormat>() {
-		@Override
-		@Nullable
-		public DateFormat convert(final String s) {
-			try {
-				if (s.equalsIgnoreCase("default"))
-					return null;
-				return new SimpleDateFormat(s);
-			} catch (final IllegalArgumentException e) {
-				Skript.error("'" + s + "' is not a valid date format. Please refer to http://docs.oracle.com/javase/6/docs/api/java/text/SimpleDateFormat.html for instructions on the format.");
-			}
-			return null;
+	private final static Option<DateFormat> dateFormat = new Option<>("date format", shortDateFormat, s -> {
+		try {
+			if (s.equalsIgnoreCase("default"))
+				return null;
+			return new SimpleDateFormat(s);
+		} catch (final IllegalArgumentException e) {
+			Skript.error("'" + s + "' is not a valid date format. Please refer to http://docs.oracle.com/javase/6/docs/api/java/text/SimpleDateFormat.html for instructions on the format.");
 		}
+		return null;
 	});
-	
-	public final static String formatDate(final long timestamp) {
+
+	public static String formatDate(final long timestamp) {
 		final DateFormat f = dateFormat.value();
 		synchronized (f) {
 			return "" + f.format(timestamp);
 		}
 	}
-	
-	final static Option<Verbosity> verbosity = new Option<Verbosity>("verbosity", Verbosity.NORMAL, new EnumParser<Verbosity>(Verbosity.class, "verbosity"))
-			.setter(new Setter<Verbosity>() {
-				@Override
-				public void set(final Verbosity v) {
-					SkriptLogger.setVerbosity(v);
+
+	final static Option<Verbosity> verbosity = new Option<>("verbosity", Verbosity.NORMAL, new EnumParser<>(Verbosity.class, "verbosity"))
+			.setter(SkriptLogger::setVerbosity);

```

Comment:
Are you sure this is functionally equivalent?

Reply:
Internally, method handles as lambdas are quite a nice piece of spaghetti and runtime code generation. But if you're sure it works, fine.